commandCenter/cleanup.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Shall we remove those files in `model`:

- velocity2d_true
- velocity3d
- velocity3d_true